CMakeLists.txt

```diff
@@ -267,6 +272,9 @@ endif()
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${WARN_FLAGS} ${SANITIZE_FLAGS} ${PROFILING_FLAGS} ${COVERAGE_FLAGS} ${LOGGING_FLAGS} ${COLORED_COMPILE}")
 message(STATUS "CMAKE_CXX_FLAGS: ${CMAKE_CXX_FLAGS}")
 
+set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} ${SANITIZE_LDFLAGS}")
```

Comment:
Is it necessary to split out the linker/compiler flags here? Or is removing -static-libasan enough?

My preference is to keep it simple and avoid complicating the code. It looks like the SANITIZER_FLAGS and SANITIZER_LDFLAGS are the same. So if the fix is to remove -static-libasan we should just do that.

Reply:
updated as per request